artifactory/commands/config_test.go

```diff
@@ -25,6 +26,20 @@ func TestBasicAuth(t *testing.T) {
 	configAndTest(t, &inputDetails)
 }
 
+func TestUsernameSavedLowercase(t *testing.T) {
+	inputDetails := config.ArtifactoryDetails{
+		Url:             "http://localhost:8080/artifactory",
+		DistributionUrl: "http://localhost:8080/distribution",
+		User:            "ADMIN", Password: "password",
+		ApiKey: "", SshKeyPath: "", AccessToken: "",
+		ServerId:  "test",
+		IsDefault: false}
+
+	outputConfig, err := configAndGetTestServer(t, &inputDetails, false)
+	assert.NoError(t, err)
+	assert.Equal(t, outputConfig.User, "admin", "The config command is supposed to save username as lowercase")
```

Comment:
outputConfig.User, "admin" should be swapped in places (the api is expected, actual)